h2olog

```diff
@@ -92,7 +92,7 @@ int trace_send_resp(struct pt_regs *ctx) {
 
 quic_bpf = """
 #define MAX_STR_LEN 32
-#define MAX_HEADER_VALUE_LEN 128
+#define MAX_HEADER_VALUE_LEN 64
```

Comment:
I've also reduced it because BPF stack size is limited to 512 bytes and 128 is sometimes too huge.